dch/DB.js

```diff
@@ -0,0 +1,58 @@
+const mongoose = require('mongoose'); //include mongose moduel 
+const Schema = mongoose.Schema;
+const mongoDB_url = 'mongodb://localhost:27017/accelration_user_data';
+
+mongoose.connect(mongoDB_url); 
+//Sucessfully connected to the databse
+mongoose.connection.on('connected', function() {
+	console.log('Connected to database');
+});
+
+//Failed the connection to the database
+mongoose.connection.on('err', function(err) {
+	console.log('Unable to connect to the databse '+ err);
+
+});
+
+mongoose.connection.on('disconnect', function(err) {
+	console.log('The database has been disconnected');
+});
+
+const accelrationDataSchema = new Schema({
+	watch_id: String, 
+	acc_x: Number, 
+	acc_y: Number, 
+	acc_z: Number, 
+	timestamp: Number
+	});
+
+module.exports.disconnect = function(){
+		mongoose.disconnect();
+};
+
+module.exports.getData = function(watch_id, startTime, stopTime, freq){
```

Comment:
that is how we are getting the message from the watch, may be we need to change it in the watch as well?